### PR TITLE
Log user id and add uuid to logs in dev

### PR DESCRIPTION
**Pull request by aub**
Fixes department-of-veterans-affairs/vets.gov-team#556

This logs the id of the user as we find their session so that we can tie the user information together with the rest of the data from the request. Since we also have the UUID appended to each bit of logging from the request, this ought to be sufficient to make the connection. I also turned on the UUID logging in dev mode just for parity, but that's not really important at all.

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -85,6 +85,7 @@ def authenticate_token
     authenticate_with_http_token do |token, _options|
       @session = Session.find(token)
       return false if @session.nil?
+      Rails.logger.info("Logged in user with id #{@session.uuid}")
```
Comment (markolson) [inline app/controllers/application_controller.rb]: I know that you can use a `proc` in log_tags to access the request - maybe we could tag part of a session or GA token from `request.cookie_jar` (or `anon` if there is none) to trace sessions, if that's a thing we need.

Comment (aub) [inline app/controllers/application_controller.rb]: @markolson I'm not sure what you mean by "maybe we could tag part of a session or GA token". I was thinking of the goal here as making it so we could connect all of the logs for a request together and to also connect them to the user so we would have the ability to dig out a series of actions for that user if needed.

Is there a need to log the GA token? 
What do you mean by "part of a session"?

Comment (markolson) [inline app/controllers/application_controller.rb]: Sorry, so.. at past jobs we've included some user-identifiable trait in the log tags themselves. Most (all?) users will have a google analytics cookie set, and any authenticated user with have an authorization token in the request header. In addition to this log message, which would appear once per request, we could add either the GA or Auth token (or whatever else) as tags (like the uuid of the request) to each _line_ of the log for the request. That would make it easier to search for all the log activity for a given user once the trait was identified.

My thinking with 'part' and 'subset' would be that if either the full authorization token or google analytics cookie is too identifiable, we could snip out a smaller substring of either of them.

**Comment by aub** [comment]
I'm going to make some changes here per Mark's comment, so please hold off on merge for now.

**Pull request closed without merging**